.github/workflows/linux_builds.yml

```diff
@@ -27,68 +29,75 @@ jobs:
             sconsflags: module_mono_enabled=yes
             bin: ./bin/godot.linuxbsd.editor.x86_64.mono
             build-mono: true
-            tests: false # Disabled due freeze caused by mix Mono build and CI
```

Comment:
Minor change but this now means we build tests on this case, which at minimum increases cache size without any benefits

Reply:
Come to think of it, I don't think we've ever re-verified if that exclusion is still relevant. I'll do a test to see if it still hangs; if it does, I'll just add `tests=no` to the flags.

Reply:
All tests passed, so that was a legacy exclusion after all!